guacamole-docker/bin/check_database.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Setting up  mysql database if not exists
+
+if [ "$SLEEP_SHORT" == "" ]; then
+	SLEEP_SHORT=5
+fi
+
+if [ "$SLEEP_LONG" == "" ]; then
+	SLEEP_LONG=10
+fi
+
+if [ "$RETRIES" == "" ]; then
+	RETRIES=5
+fi
```

Comment:
If new environment variables are to be supported by this image, the naming should be more specific. `SLEEP_SHORT`, `SLEEP_LONG`, and `RETRIES` are too broad.